### PR TITLE
A new keyword "Last REST Instance" is implemented

### PR DESCRIPTION
**Pull request by TeddyTeddy**
This keyword allows the very last instance to be written to a file.
This becomes handy, when you just made a request and you want to get the instance for it.

A sample usage would be:

```
*** Keywords ***
Gather Instances While Creating Parameterized Resources
    [Arguments]     ${record list}         ${private token}
    FOR     ${record}    IN      @{record list}
        POST    /projects       body=${record}[json_resource]    headers=${private token}
        Last Rest Instance      file_path=${record}[on_create_instance]      sort_keys=True     # instance for the POST request
    END
```

### CODE REVIEW

**Comment by asimell** [comment]
@TeddyTeddy  This functionality seems to exist already? You can give a `file_path` argument to `Output` keyword, which then writes your output into a file.

**Comment by TeddyTeddy** [comment]
@asimell , as per your comment:

> @TeddyTeddy This functionality seems to exist already? You can give a `file_path` argument to `Output` keyword, which then writes your output into a file.

Hi! Yes, [output keyword](https://github.com/asyrjasalo/RESTinstance/blob/93a4811648fe9606060fe116fc0865d153d5b92a/src/REST/keywords.py#L1078) does write the request & response schema to a file, but it does not write the whole instance. The instance is a larger entity, which is composed of:

- request in JSON
- response in JSON
- the contract: the request schema + the response schema

If we want to have the instance as a record into a file, this utility keyword "Last REST Instance" would be helpful.

**Comment by asyrjasalo** [comment]
I think @asimell is correct. There is a way to achieve above without adding a new keyword.

File: src/REST/keywords.py [hunk 1]
```diff
@@ -1355,9 +1402,8 @@ def _instantiate(self, request, response, validate_schema=True):
                 self._validate_schema(request_properties, request)
             if response_properties:
                 self._validate_schema(response_properties, response)
-        request_properties["body"] = self._new_schema(request["body"])
-        request_properties["query"] = self._new_schema(request["query"])
-        response_properties["body"] = self._new_schema(response["body"])
+        self._form_sub_schema(request_properties, request)
```
Comment (TeddyTeddy) [inline src/REST/keywords.py:1405]: @asyrjasalo @asimell Sure, you can opt out my "Last REST Instance" keyword; but one change here that you might need to take into consideration is that before my changes only request & response body were outputted, but with my changes every key/value pair in the request & response are now outputted via "Last REST Instance" keyword.

Also, the end user of the library would have to write & maintain a function containing at least 10 lines of code, while 
the library itself has the instance data already ready to be written into a file. Imagine that tens of users does the same. Isn't that a wasteful effort from the end users' perspective? 

I have nothing further to say. Thanks anyway bringing life back to this important library.

Comment (asimell) [reply src/REST/keywords.py:1405]: > only request & response body were outputted

Incorrect. If you call `Output` without any parameters it outputs the full request and response object.

There's also `Output Schema` to output the schema only and you can give a `file_path` argument to that keyword as well.

**Pull request closed without merging**